content/_index.md

```diff
@@ -4,14 +4,14 @@ sort_by = "weight"
 +++
 # Overview
 
-**PeopleForBikes** is an organization promoting cycling in the US.
+**[PeopleForBikes](https://www.peopleforbikes.org/)** is an industry coalition made up of 280 cycling-industry members and a cycling community of almost 1.4 million individual riders to promote stronger representation in congress and coordinated advocacy programs and  help develop and promote economic policies that benefit bike businesses.
 
-**PeopleForBikes Tech** is the technical branch of it. We aim at simplifying and
-automating tasks to help our advocates focus on what they do best: get more
+**[PeopleForBikes Tech](http://example.com/)** is PFB's technical arm. We aim to simplify and
+automate tasks to help our advocates focus on what they do best: get more
 people riding bikes more often, and make bike riding better for everyone.
 
 Our projects:
 
-* retrieve: Simplify downloading PeopleForBikes dataset
-* svggloo: Render SVG templates
-* measure: Quantify and identify bike lanes in cities
+* [retrieve](http://example.com/): Simplify downloading PeopleForBikes dataset
+* [svggloo](http://example.com/): Render SVG templates
+* [measure](http://example.com/): Quantify and identify bike lanes in cities
```

Comment:
The text sounds much better 👍 

The URLs should point to the GitHub repos.

Reply:
Yay! I've updated all 3, but did not know which the `measure` repo should point to, so I pointed to the list of repos.